### PR TITLE
Add support for LESS and include current CSS as is for now

### LINKED ISSUE #778: Moving from vanilla css to less
Currently the Makefile includes a directive for `make css` which runs

```
        mkdir -p static/build
        bash static/css/all.cssh > static/build/all.css
```

Low hanging fruit to migrate this to use `less`

### PR DESCRIPTION
**Pull request by html5cat**
Added `sass` support to the repo. Fixes #778 
For now, within the VM you need to run `gem install sass`.
Updated `make css` to work as before.
Fixed a couple of invalid css selectors (thanks sass!).

### CODE REVIEW

**Comment by mekarpeles** [comment]
hooray! I'll pull onto https://dev.openlibrary.org and see if anything breaks when we try to build. Also, looking into python `libsass-python` (i.e. pip install `libsass`) to simplify the build process a la http://sass-lang.com/libsass

**Comment by html5cat** [comment]
Totally, I saw `libsass`, but wasn't sure how it would fit in the codebase, i.e. where would you put the compile scripts. Sass gem also can do `watch` to live update for dev, which I think might be a hassle with `libsass`.

**Comment by mekarpeles** [comment]
unfortunately our serves have ruby 1.9 so we'll have to update ruby to even install the gem

**Comment by mekarpeles** [comment]
It looks like we have several places which still need to be resolved:

e.g. https://github.com/html5cat/openlibrary/blob/01e522fb1aa3c36e31dcfabca57d60b4bf0fc55f/static/css/_datatables.scss#L83 	`* width: 50px;`

**Comment by html5cat** [comment]
Fixed css typos and removed repeating blocks from master.scss that are in other files

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -37,6 +38,11 @@ python-yaml"
 
 apt-get install -y $APT_PACKAGES
 
+# Install `lessc` dependencies for css pre-processing
+npm install -g less
+npm update -g less
```
Comment (mekarpeles) [inline scripts/bootstrap.sh]: I think this is unnecessary; will remove in a followup which addresses i18n vagrant issues

**Pull request merged**